tsl/test/sql/transparent_decompression_ordered_index.sql.in

```diff
@@ -0,0 +1,132 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+\set TEST_BASE_NAME transparent_decompression
+SELECT format('include/%s_query.sql', :'TEST_BASE_NAME') AS "TEST_QUERY_NAME",
+    format('%s/results/%s_results_uncompressed.out', :'TEST_OUTPUT_DIR', :'TEST_BASE_NAME') AS "TEST_RESULTS_UNCOMPRESSED",
+    format('%s/results/%s_results_compressed.out', :'TEST_OUTPUT_DIR', :'TEST_BASE_NAME') AS "TEST_RESULTS_COMPRESSED" \gset
+
+SELECT format('\! diff %s %s', :'TEST_RESULTS_UNCOMPRESSED', :'TEST_RESULTS_COMPRESSED') AS "DIFF_CMD" \gset
+
+-- Testing Index Scan backwards ----
+-- We want more than 1 segment in atleast 1 of the chunks
+
+CREATE TABLE metrics_ordered_idx (
+    time timestamptz NOT NULL,
+    device_id int,
+    device_id_peer int,
+    v0 int
+);
+
+SELECT create_hypertable ('metrics_ordered_idx', 'time', chunk_time_interval => '2days'::interval);
+
+ALTER TABLE metrics_ordered_idx SET (timescaledb.compress, timescaledb.compress_orderby = 'time ASC', timescaledb.compress_segmentby = 'device_id,device_id_peer');
+
+INSERT INTO metrics_ordered_idx (time, device_id, device_id_peer, v0)
+SELECT time,
+    device_id,
+    0,
+    device_id
+FROM generate_series('2000-01-13 0:00:00+0'::timestamptz, '2000-01-15 23:55:00+0', '15m') gtime (time),
+    generate_series(1, 5, 1) gdevice (device_id);
+
+INSERT INTO metrics_ordered_idx (time, device_id, device_id_peer, v0)
+SELECT generate_series('2000-01-20 0:00:00+0'::timestamptz, '2000-01-20 11:55:00+0', '15m'),
+    3,
+    3,
+    generate_series(1, 5, 1);
+
+INSERT INTO metrics_ordered_idx (time, device_id, device_id_peer, v0)
+SELECT generate_series('2018-01-20 0:00:00+0'::timestamptz, '2018-01-20 11:55:00+0', '15m'),
```

Comment:
same here - should be 10s.

Reply:
even with the reduced dataset plan shapes don't change

Reply:
in the shared tests there are hypertables with 4k tuples per device per chunk which is where queries should live that need more data

Reply:
The data set matters here as the execution path is different when we have compressed data with multiple segments.  Since that requires a larger data set, we should move the test to the shared test.  Could you add an issue so that we don't lose track of it and moving the test can be done in a subsequent PR?